### ISSUE TITLE
Custom styles for file Input

### ISSUE THREAD

**Issue by btzr-io** [btzr-io/Villain#138]
Add custom styles for the file input component on the demo example.
> Note: it should be simple ( minimalist ) and not affect usability or accessibility

**Comment by kamilszmajdzinski** [comment]
@btzr-io Created custom file input
Raised PR https://github.com/btzr-io/Villain/pull/146
